webapp/stores/browser_store.jsx

```diff
@@ -160,10 +161,14 @@ class BrowserStoreClass {
         if (landingPageSeen) {
             this.setLandingPageSeen(landingPageSeen);
         }
+
+        if (selectedTeams) {
+            this.setItem('selected_teams', selectedTeams);
+        }
     }
 
     isLocalStorageSupported() {
-        if (this.checkedLocalStorageSupported !== '') {
+        if (this.checkedLocalStorageSupported !== undefined) { // eslint-disable-line no-undefined
```

Comment:
I think the proper way to null check in javascript is `if (variable != null) { // do stuff }`